### PR TITLE
Update technical-preview-free-tier.md

### PR DESCRIPTION
**Pull request by danieltprice**
Update free tier limit content for the branching release.
https://websitemain62807-dpriceupdatefreetiercontent.gatsbyjs.io/docs/introduction/technical-preview-free-tier/

### CODE REVIEW

File: content/docs/reference/technical-preview-free-tier.md [hunk 1]
```diff
@@ -5,49 +5,27 @@ enableTableOfContents: true
...
+The Technical Preview Free Tier has the following limits:
 
-- Project data size is limited to 10GB
-- The Point in Time Reset (PITR) window is limited to 7 days of _reasonable usage_
-- The compute node is limited to 1 vCPU and 256MB of RAM
+- 1 Neon project
+- 10 branches
+- 3 GB of data per branch
+- Up to 3 compute nodes (endpoints), one per branch
+- 1 vCPU and 256MB of RAM per compute node
```
Comment (stepashka) [inline content/docs/reference/technical-preview-free-tier.md:14]: @kelvich , could you verify this?

File: content/docs/reference/technical-preview-free-tier.md [hunk 2]
```diff
@@ -5,49 +5,27 @@ enableTableOfContents: true
...
+Neon stores data in its own internal format. The 10GB data size limit in the Technical Preview free tier applies to the logical size of your Neon project. The logical size is the sum of all database sizes in the branches of your project. For information about checking the data size of your Neon project, see [../../manage/projects](Projects).
 
 <a id="#point-in-time-reset/"></a>
 
 ## Point in Time Reset
 
 Neon storage consumes extra space in order to support Point in Time Reset (PITR) and the ability to reset a branch to a historical state. Historical data is stored in log-based format.
 
-Neon limits the amount of modification history that is stored in the Technical Preview Free Tier.
+Neon limits the amount of modification history that is stored in the Technical Preview Free Tier, as described above.
```
Comment (stepashka) [inline content/docs/reference/technical-preview-free-tier.md:31]: not sure what does the 'as described above' relate to.
we will introduce some form of the history limit, but it's not currently in place..

Comment (danieltprice) [reply content/docs/reference/technical-preview-free-tier.md:31]: Remove "as describe above".

File: content/docs/reference/technical-preview-free-tier.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/reference/technical-preview-free-tier.md:14]: Changed to 512MB as per this comment in Slack: https://neondb.slack.com/archives/C033A2WE6BZ/p1669840130644329?thread_ts=1669378336.698969&cid=C033A2WE6BZ

File: content/docs/introduction/technical-preview-free-tier.md [hunk 3]
```diff
@@ -7,44 +7,22 @@ redirectFrom:
...
 
-- Project data size is limited to 10GB
-- The Point in Time Reset (PITR) window is limited to 7 days of _reasonable usage_
-- The compute node is limited to 1 vCPU and 256MB of RAM
+- 1 Neon project
+- 10 branches
+- 3 GB of data per branch
+- 3 compute nodes (endpoints)
+- 1 vCPU with a limit of 4GB of RAM per compute node (with 512MB of RAM allocated to PostgreSQL shared buffers)
+- A point-in-rime reset (PITR) window of 7 days of _reasonable usage_
```
Comment (ololobus) [inline content/docs/introduction/technical-preview-free-tier.md:17]:
```suggestion
- A point-in-time recovery (PITR) window of 7 days of _reasonable usage_
```

File: content/docs/introduction/technical-preview-free-tier.md [hunk 4]
```diff
@@ -7,44 +7,22 @@ redirectFrom:
...
-Neon separates storage and compute and stores data in its own internal format.
-The 10GB data size limit applies to the logical size of a Neon project. The logical size is the sum of all relation sizes in the project.
-
-To check the logical size of the databases in your Neon project, run the following query:
-
-```sql
-SELECT pg_size_pretty(sum(pg_database_size(datname)))
-FROM pg_database;
-```
-
```
Comment (ololobus) [inline content/docs/introduction/technical-preview-free-tier.md:31]: This command works well for me on our compute, maybe we can keep it?

Comment (danieltprice) [reply content/docs/introduction/technical-preview-free-tier.md:31]: @ololobus Does this query make sense now that we have moved our data size limits to branches (3GB per branch)? I am assuming this query gives you a project-wide sum.

Comment (ololobus) [reply content/docs/introduction/technical-preview-free-tier.md:31]: > I am assuming this query gives you a project-wide sum.

No, it gives somewhat the logical branch (timeline) size, IIUC. Actually, I think that it's impossible to give the actual project (tenant) size from the running compute. This info is only available on the pageserver

cc @lubennikovaav @LizardWizzard please, correct me if I'm wrong

Comment (danieltprice) [reply content/docs/introduction/technical-preview-free-tier.md:31]: OK. I have added it back to the docs, but I have moved it to this branches page, which will be published soon with https://github.com/neondatabase/website/pull/336.
https://websitemain62807-dpricemanageusers.gatsbyjs.io/docs/manage/branches/#check-the-data-size

Comment (LizardWizzard) [reply content/docs/introduction/technical-preview-free-tier.md:31]: > No, it gives somewhat the logical branch (timeline) size, IIUC. Actually, I think that it's impossible to give the actual project (tenant) size from the running compute. This info is only available on the pageserver

Sounds correct

**Pull request merged**